Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UpgradeData.xml

```diff
@@ -11883,6 +11910,8 @@
         <EffectArray Reference="Unit,AP_TorrasqueCorpse,Food" Value="2"/>
         <EffectArray Operation="Subtract" Reference="Unit,AP_TorrasqueChrysalis,CostResource[Minerals]" Value="100"/>
         <EffectArray Reference="Unit,AP_TorrasqueChrysalis,Food" Value="2"/>
+        <EffectArray Operation="Subtract" Reference="Unit,AP_Tyrannozor,CostResource[Minerals]" Value="200"/>
+        <EffectArray Reference="Unit,AP_Tyrannozor,Food" Value="3"/>

```

Comment:
You'll want to include the burrowed Tyrannozor in this too, otherwise it'll cost food to burrow

Reply:
Fixed, thanks for catching

Reply:
![Snímek obrazovky pořízený 2024-07-26 19-55-23](https://github.com/user-attachments/assets/fe342a7a-cb86-4294-bf23-9c4a6bd933e2)
Minerals too

Reply:
Fixed